lib/active_shipping/carriers/ups.rb

```diff
@@ -612,6 +612,15 @@ def build_international_forms(xml, origin, destination, packages, options)
           xml.InvoiceNumber(options[:invoice_number])
           xml.Comments(options[:comments])
 
+          xml.FreightCharges do
```

Comment:
Is this okay to only show on orders with `options[:international]`?

Reply:
Commercial invoices are only generated for international orders AFAIK, so yes this should be what we want. 
This is also the logic that is currently being used for all of our UPS commercial invoices too, so I would imagine that if this was causing an issue we'd know by now.

Reply:
Ah, so the check for `options[:international]` isn't really all that necessary?

Reply:
From what I can tell it's unnecessary, but I also don't see any issues with it. It's been there since 2017 so I'd rather not mess with it if it's not causing any issues

Reply:
Heard.